cert/test_files/google.pem

```diff
@@ -0,0 +1,50 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It could be useful to add a README in the `test_files` directory explaining where each of these certs came from - I see that this one expires next month, so will the tests start failing once that time arrives? 

If you just grabbed this from google.com, then bonus points for a `generate_test_files.sh` script that uses openssl or something to go grab the latest cert and write it to this file, although it's arguably overkill 😄

Reply:
These tests don't check for certificate expiry, so if the cert expires it shouldn't change the outcome of the test results. But I'll add a little readme to make this clearer!